### PR TITLE
[RNMobile] Improve autosave on mobile and ping the native at each keystroke

### PR DESCRIPTION
**Pull request by daniloercoli**
This is a quick patch that improves the autosave for mobile, and introduces pings to the native side at each keystroke. The debouncing logic that handles the actual saving of the content on the local DB is already available in apps, and already used in the other editors included in the apps.

Note: This is just a way to override default behavior for mobile, and make it pings the native side at each keystroke. 

_We're still working on an old version of `master`, and the local autosave added [here](https://github.com/WordPress/gutenberg/pull/16490) is still not available. We should  probably revisit the autosave when we switch back to use `master`_.

### CODE REVIEW

**Review by etoledom — APPROVED** [review]
Tested on iOS and works as expected, sending the autosave event on every content change 🎉 
Let's wait for @maxme review before merging 🙂

**Comment by maxme** [comment]
I just tested it in wpandroid by adding a debug log [here](https://github.com/wordpress-mobile/WordPress-Android/blob/c02c3168ec394edfb758f60dba19217da3e39dac/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L2289). It gets called at each keystroke and the event is correctly debounced later in the app. I also did the same test with Aztec to compare implementations and I got the same results. 🎉

**Pull request merged**
